src/test/java/racingcar/domain/RacingCarsTest.java

```diff
@@ -93,12 +93,13 @@ void one_winner() {
     void several_winners() {
         //given
         List<Car> cars = new ArrayList<>();
+        RunStrategy runStrategy = () -> true;
 
-        Car car1 = new Car("1", ((min, max) -> 9));
+        Car car1 = new Car("1", runStrategy);
         car1.run(1);
-        Car car2 = new Car("2", ((min, max) -> 9));
+        Car car2 = new Car("2", runStrategy);
         car2.run(3);
-        Car car3 = new Car("3", ((min, max) -> 9));
+        Car car3 = new Car("3", runStrategy);
         car3.run(3);
 
         cars.add(car1);
```

Comment:
`Arrays.asList()`를 활용하여 코드를 줄여볼 수 있을 것 같습니다.
```suggestion
        List<Car> cars = Arrays.asList(car1, car2, car3);
```